examples/ownable-two-step/src/lib.rs

```diff
@@ -0,0 +1,35 @@
+#![cfg_attr(not(test), no_main)]
+extern crate alloc;
+
+use alloc::vec::Vec;
+
+use alloy_primitives::{Address, U256};
+use openzeppelin_stylus::{
+    access::ownable_two_step::Ownable2Step,
+    token::erc20::{Erc20, IErc20},
+};
+use stylus_sdk::prelude::{entrypoint, public, sol_storage};
+
+sol_storage! {
+    #[entrypoint]
+    struct Ownable2StepExample {
+        #[borrow]
+        Erc20 erc20;
+        #[borrow]
+        Ownable2Step ownable;
+    }
+}
+
+#[public]
+#[inherit(Erc20, Ownable2Step)]
+impl Ownable2StepExample {
+    pub fn transfer(
+        &mut self,
+        to: Address,
+        value: U256,
+    ) -> Result<(), Vec<u8>> {
+        self.ownable._ownable.only_owner()?;
```

Comment:
I decided to not add again `only_owner` as a function of the `Ownable2Step` contract, since anyone inheriting from it can still access it and we are not making it public anymore. I do understand that this way of accessing is not the most ergonomic, but I still think it is better than just re-implementing it. But open to suggestions of course :)